blog/_posts/2021-01-19-scala-native-0.4-release.md

```diff
@@ -0,0 +1,225 @@
+---
+layout: blog-detail
+post-type: blog
+by: Wojciech Mazur, Scala Center
+title: A sneak peek into the Scala Native 0.4 release
+---
+
+If you ever thought about running your perfect Scala code in an environment other than JVM, you've probably already met
+Scala Native---a project with a promise of instant startup, low memory usage, interop with C libraries, and competitive performance
+thanks to [Interflow optimizer](https://scala-native.readthedocs.io/en/latest/blog/interflow.html).
+
+However, since there have been no release in over a year, you may ask yourself---is this project even alive?<br>
+With the long-awaited release 0.4.0 it proves it is! 
+Today we are going to discuss the most impacting new features and changes coming in this new major release.
+We will start with changes according to Scala Native compiler plugin, especially supported Scala versions and cross-platform testing.
+Next we will talk about changes to C interoperability and SBT build that need to be taken into account since when working this release.
+Last, but not least we will discuss why we needed to break binary compatibility with previous versions.
+
+## Support for latest Scala 2 versions
+Until now Scala Native supported only Scala 2.11, which is now perceived by most of the developers as deprecated.
+We can proudly announce that Scala Native is now compatible with Scala versions 2.12.13 and 2.13.4, in addition to the 2.11.12 with known bugs fixed.
```

Comment:
I think this information should come sooner in the blog post. Like really really soon, perhaps in the first sentence or even the title. By far the single most important information in the entire article is "Scala Native supports Scala 2.12 and 2.13 now". I suggest shouting this from the rooftops.